Tests/Specs/unified-test-format/tests/valid-fail/entity-client-apiVersion-unsupported.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I should have synced this file back when implementing versioned API but I neglected to ( we can't actually run this anyway, but it seemed good to catch up)